# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../dbus/hotfix-syslog_h.patch
# Copyright (C) 2008 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- dbus-1.2.10/dbus/dbus-sysdeps-util-unix.c.orig	2008-12-19 20:35:23.000000000 +0100
+++ dbus-1.2.10/dbus/dbus-sysdeps-util-unix.c	2008-12-27 17:10:34.000000000 +0100
@@ -39,6 +39,7 @@
 #include <errno.h>
 #include <fcntl.h>
 #include <sys/stat.h>
+#include <sys/syslog.h>
 #include <grp.h>
 #include <sys/socket.h>
 #include <dirent.h>
